### PR TITLE
Control flow: add quizzes

### LINKED ISSUE #215: Control flow

### PR DESCRIPTION
**Pull request by despadam**
Fixes #215

### CODE REVIEW

File: tutorial/control_flow.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+                    x = 0
+                    for i in range(10):
+                        for j in range(-1, -10, -1):
+                            x += 1
+                    print(x)
+                </code></pre>
+            """,
+            options={
+                "9": "Pay attention to the range of the outter loop.",
+                "18": "Correct!",
```
Comment (yakutovicha) [inline tutorial/control_flow.py:104]: Please revisit this question. Something is here.
```suggestion
                "90": "Correct!",
```

File: tutorial/control_flow.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+                "20": "Remember that <code>range()</code> does not include its stop element.",
+            },
+            correct_answer="18",
```
Comment (yakutovicha) [inline tutorial/control_flow.py:107]:
```suggestion
            correct_answer="90",
```

File: tutorial/control_flow.py [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+            correct_answer="Yes",
+            shuffle=True,
+        )
+
+        q5 = Question(
+            question="""How do you check for multiple conditions in a single if statement?""",
+            options={
+                "if x > 10 and y < 5:": "Correct! Python uses the keyword <strong>and</strong>.",
+                "if x > 10 && y < 5:": "This is not a valid Python operator.",
+                "if x > 10 & y < 5:": "This is not a valid Python operator.",
```
Comment (yakutovicha) [inline tutorial/control_flow.py:81]:
```suggestion
                "if x > 10 & y < 5:": "& is bitwise AND operator.",
```

**Review by yakutovicha — CHANGES_REQUESTED** [review]
Thanks, @despadam. I had some fun answering the quizzes.

File: tutorial/control_flow.py (hunk 1 above)
Comment (despadam) [reply tutorial/control_flow.py:104]: > Please revisit this question. Something is here.

Good catch! I was using the wrong range in the outer loop. It is fixed now

**Review by yakutovicha — APPROVED** [review]
Looks good!

**Pull request merged**